library/decl_kinds.ml

```diff
@@ -12,6 +12,8 @@
 
 type discharge = DoDischarge | NoDischarge
 
+type check_guard = bool option
```

Comment:
I don't like `bool option`. Why don't you create a three-valued type like is done around? Why do you need three values BTW?